### ISSUE TITLE
Saturated results with UPSCALE IMAGE node

### ISSUE THREAD

**Issue by sujyQ** [chaiNNer-org/chaiNNer#2105]
**Information:**

- Chainner version: 0.19.2
- OS: Windows 10

**Description**
With MM-RealSRGAN model, superresolved results are somewhat saturated. but with Real-ESRGAN, there was no problem.
What can i do or would it be solved?
![image](https://github.com/chaiNNer-org/chaiNNer/assets/82137849/9e3e97e6-e2b7-42b8-8486-5800b9535128)
![image](https://github.com/chaiNNer-org/chaiNNer/assets/82137849/6576d1f3-d23e-4495-8e7a-be2b77517b2f)

**Logs**
[logs.zip](https://github.com/chaiNNer-org/chaiNNer/files/12357307/logs.zip)

**Comment by RunDevelopment** [comment]
I don't think we support [MM-RealSR](https://github.com/TencentARC/MM-RealSR). It's even wrongly detected as an ESRGAN model. 

We have to explicitly add support for every model architecture (for PyTorch), so there will always be some that we don't support.

**Comment by sujyQ** [comment]
I got it. I thought MM-RealSR was kind of ESRGAN variants, as it uses Real-ESRGAN as backbone.

**Comment by joeyballentine** [comment]
It probably wouldn't be too difficult to add support for it. I'll consider it
